cloud

```diff
@@ -624,7 +632,15 @@ function cloud_functions_deploy() {
                           --entry-point kcidb_load_queue \
                           --env-vars-file "$env_yaml" \
                           --runtime python37 \
-                          --trigger-topic "${trigger_topic}" \
+                          --trigger-topic "${load_queue_trigger_topic}" \
+                          --memory 256MB \
+                          --timeout 540
+    cloud_function_deploy "$project" "${prefix}pick_notifications" \
+                          "$source" \
+                          --entry-point kcidb_pick_notifications\
+                          --env-vars-file "$env_yaml" \
+                          --runtime python37 \
+                          --trigger-topic "${pick_notifications_trigger_topic}" \
```

Comment:
Could you move this function deployment before the `send_notifications` one? This way we'll be replacing functions back-to-front (pipeline-wise), making it easier to preserve compatibility with updates.